File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid-font/IntentServices/BALANCA/Commands/ConfigurarModeloBalanca.cs
using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.BALANCA.Commands
{
    class ConfigurarModeloBalanca : BalanceCommand
    {
        readonly int modeloBalanca;

        public ConfigurarModeloBalanca(int modeloBalanca) : base("ConfigurarModeloBalanca")
        {
            this.modeloBalanca = modeloBalanca;
        }

        protected override string FunctionParameters()
        {
            return "\"modeloBalanca\"" + ":" + this.modeloBalanca;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/BALANCA/ConfigurarModeloBalanca.ts
import {BalancaCommand} from './BalancaCommand';

export class ConfigurarModeloBalanca extends BalancaCommand {
  readonly modeloBalanca: number;

  constructor(modeloBalanca: number) {
    super('ConfigurarModeloBalanca');
    this.modeloBalanca = modeloBalanca;
  }

  functionParameters(): object {
    return {
      modeloBalanca: this.modeloBalanca,
    };
  }
}
